lib/MTT/Test/Run.pm

```diff
@@ -525,6 +525,12 @@ sub _do_run {
     $tmp = $ini->val($section, "treat_timeouts_as_fail");
     $config->{treat_timeouts_as_fail} = $tmp
         if (defined($tmp));
+    $config->{corefile_size_limit} = -1;
+    $tmp = $ini->val($section, "corefile_size_limit");
+    $config->{corefile_size_limit} = $tmp
+        if (defined($tmp));
+
+    MTT::DoCommand::set_corefile_size_limit($config->{corefile_size_limit});
```

Comment:
I'm reading the [perl manual page](http://search.cpan.org/~jhi/BSD-Resource-1.2910/Resource.pm#ERRORS) for `BSD::Resource` and the notes under **Errors** and **Known Issues** are a bit concerning with environment support for `setrlimit`.

In the case where the INI file does not specify `corefile_size_limit` then we should not be calling `MTT::DoCommand::set_corefile_size_limit`. This allows folks with unsupported environments to still run MTT, but without this feature.

I suggest changing this line to:
```perl
MTT::DoCommand::set_corefile_size_limit($config->{corefile_size_limit})
   if (defined($tmp));
```

or, slightly better,
```perl
if (defined($tmp)) {
    $config->{corefile_size_limit} = $tmp;
    MTT::DoCommand::set_corefile_size_limit($config->{corefile_size_limit});
}
```